pkg/ansible/run.go

```diff
@@ -275,18 +275,25 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string, gvks []schema.GroupVers
 // getAnsibleDebugLog return the value from the ANSIBLE_DEBUG_LOGS it order to
 // print the full Ansible logs
 func getAnsibleDebugLog() bool {
-	const envVar = "ANSIBLE_DEBUG_LOGS"
+	envVars := []string{"ANSIBLE_DEBUG_LOGS", "ANSIBLE_ENABLE_TASK_DEBUGGER"}
```

Comment:
The `ANSIBLE_DEBUG_LOGS` will allow you to check the ansible logs in the operator container.
see that it is the func `getAnsibleDebugLog()`.So, if you would like to re-use is missing; rename the func name and iits comment as well.